src/components/ErrorPanel/index.jsx

```diff
@@ -0,0 +1,82 @@
+import { Component } from 'react';
+import { instanceOf, oneOfType, string } from 'prop-types';
+import classNames from 'classnames';
+import Markdown from 'react-markdown';
+import { withStyles } from '@material-ui/core/styles';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import Paper from '@material-ui/core/Paper';
+import ChevronDownIcon from 'mdi-react/ChevronDownIcon';
+import ErrorBox from './ErrorBox';
+
+@withStyles(theme => ({
+  paper: {
+    padding: `0 ${theme.spacing.double}px`,
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  pad: {
+    paddingTop: 9,
+    paddingBottom: 9,
+  },
+  error: {
+    backgroundColor: theme.palette.error.dark,
```

Comment:
The dark theme values come from taskcluster-web's dark theme. You'll want to change these to the values of the current theme if the color scheme doesn't look quite right here.

Reply:
This still needs to be done.